### ISSUE TITLE
only integer handling?

### ISSUE THREAD

**Issue by jhiza** [fauxsoup/estatsd_server#1]
Do I have something configured incorrectly or am I seeing that only integer values are supported?

```
=ERROR REPORT==== 12-Dec-2014::11:17:18 ===
** Generic server estatsd_listener terminating 
** Last message in was {udp,#Port<0.15456>,
                            {10,8,16,76},
                            38002,
                            <<"metric.latency:0.673055648803711|ms\n">>}
** When Server state == #Port<0.15456>
** Reason for termination == 
** {badarg,[{erlang,list_to_integer,["0.673055648803711"],[]},
            {estatsd_listener,parse_value,2,
                              [{file,"src/estatsd_listener.erl"},{line,59}]},
            {lists,foreach,2,[{file,"lists.erl"},{line,1336}]},
            {estatsd_listener,handle_info,2,
                              [{file,"src/estatsd_listener.erl"},{line,33}]},
            {gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,593}]},
            {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,659}]},
            {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]}
```

**Comment by soup-in-boots** [comment]
Sorry for the late reply.

Yes, at this time only integers are supported. However, the restriction to integers should really only apply for counter values (owing to the way we store and update the counters). It also only appears to be a limitation for the UDP protocol. I'll have a patch for this soon (probably tonight).

Also, consider using "microsecond" (1000 per millisecond) values for your timers if you absolutely cannot wait.

**Comment by soup-in-boots** [comment]
Fix for this is in v1.3.5. Give it a try.
